client/boot/locale.js

```diff
@@ -23,5 +24,26 @@ export const setupLocale = ( currentUser, reduxStore ) => {
 		reduxStore.dispatch( setLocale( currentUser.localeSlug, currentUser.localeVariant ) );
 	}
 
+	if ( '__requireChunkCallback__' in window ) {
```

Comment:
@Automattic/team-calypso: Do we have any guidance on adding new globals?

Reply:
@sgomes No real guidance, though this seems mostly OK. Maybe we should consider namespacing it, like `__a8c_requireChunkCallback__`, but that's just an abundance of caution.